### PR TITLE
feat(registration validation): add status translations

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Add translations

## Why

UI Improvements

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/assets/locales/de/main.json [hunk 1]
```diff
@@ -137,7 +137,9 @@
         "BUSINESS_PARTNER_NUMBER": "BPN Creation"
       },
       "status": {
+        "IN_PROGRESS": "In Progress",
         "TO_DO": "To Do",
+        "DONE": "Done",
         "FAILED": "Failed"
```
[the remark this answers is not in the document]
Comment (nidhigarg-bmw) [reply src/assets/locales/de/main.json]: @evegufy Added translations after discussion with Julia

File: src/assets/locales/de/main.json [hunk 2]
```diff
@@ -137,8 +137,10 @@
         "BUSINESS_PARTNER_NUMBER": "BPN Creation"
       },
       "status": {
-        "TO_DO": "To Do",
-        "FAILED": "Failed"
+        "IN_PROGRESS": "Im Gange",
+        "TO_DO": "Machen",
```
Comment (jjeroch) [inline src/assets/locales/de/main.json:141]: Please rename TO_DO to "Ausstehend"

File: src/assets/locales/de/main.json [hunk 3]
```diff
@@ -137,8 +137,10 @@
         "BUSINESS_PARTNER_NUMBER": "BPN Creation"
       },
       "status": {
-        "TO_DO": "To Do",
-        "FAILED": "Failed"
+        "IN_PROGRESS": "Im Gange",
```
Comment (jjeroch) [inline src/assets/locales/de/main.json:140]: Please rename IN_PROGRESS to "In Bearbeitung"

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: src/assets/locales/de/main.json (hunk 3 above)
Comment (nidhigarg-bmw) [reply src/assets/locales/de/main.json]: @jjeroch Made the changes

**Review by jjeroch — APPROVED** [review]

**Review by lavanya-bmw — APPROVED** [review]

**Review by oyo — APPROVED** [review]

**Pull request merged**